Swap left and right double tap key event handlers

alphastark opened the pull request:
Swap left and right double tap key event handlers for left and right keyboard arrow keys.


  Swapped keyEvent listeners for left and right keyboard arrow keys as they were swapped in the code causing the opposite of the desired behavior.
  Fixes issue #1947

alphastark committed:
Swap left and right double tap key event handlers

Swap left and right double tap key event handlers for left and right keyboard arrow keys.

Secozzi reviewed (changes requested):
Add a changelog entry

alphastark committed:
Merge branch 'aniyomiorg:main' into patch-1

alphastark committed:
Added fix keyEvent listener swap for arrow keys

Added fix: swapped the keyEvent listeners for left and right keyboard arrow keys to correct behavior.

alphastark reviewed (commented):
Added CHANGELOG entry

alphastark committed:
Added fix keyEvent listener swap for arrow keys

Added fix: swapped the keyEvent listeners for left and right keyboard arrow keys to correct behavior.

alphastark committed:
Added fix keyEvent listener swap for arrow keys

Added fix: swapped the keyEvent listeners for left and right keyboard arrow keys to correct behavior.
